src/Utilities/results.jl

```diff
@@ -88,9 +88,17 @@ function _dual_objective_value(
     )
 end
 
+_lower(set::MOI.Interval, ::Type) = set.lower
+_upper(set::MOI.Interval, ::Type) = set.upper
+_lower(::MOI.ZeroOne, ::Type{T}) where {T} = zero(T)
+_upper(::MOI.ZeroOne, ::Type{T}) where {T} = one(T)
+
 function _dual_objective_value(
     model::MOI.ModelLike,
-    ci::MOI.ConstraintIndex{<:MOI.AbstractScalarFunction,<:MOI.Interval},
+    ci::MOI.ConstraintIndex{
+        <:MOI.AbstractScalarFunction,
+        <:Union{MOI.ZeroOne,MOI.Interval},
+    },
```

Comment:
This needs to be a separate PR. What is `ZeroOne` doing in a PR about testing constraint duals?

Reply:
This is an issue detected by running the dual tests through all bridges. This is therefore tested in this PR as well

Reply:
The `AbstractFunctionConversionBridge` already supports setting `ConstraintDual`, they treat it the same as `ConstraintDualStart` so it got hit by the new tests.

> How can there be a dual if the ZeroOne set is involved?

From the perspective of `_dual_objective_value`, since `ZeroOne` is equivalent to `Interval` + `Integer`, and `Integer` is ignored by `_dual_objective_value` then it's ok to treat `ZeroOne` as an `Interval`. If the solver doesn't have any dual, we will fail when we query the dual from the solver anyway with the expected error. Here, at the level of bridges or `_dual_objective_value`, we are just taking the transpose of linear maps so we shouldn't worry about whether some variables are integer or not.

Reply:
We can disable the tests for the failing ones yes